.github/workflows/docs.yml

```diff
@@ -0,0 +1,137 @@
+name: Documentation workflow
+
+# The event triggers are configured as following:
+# - on branch main, trigger the workflow on every push
+# - on any pull request, trigger the workflow
+# This is to avoid running the workflow twice on pull requests.
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  build_and_deploy_docs:
+    runs-on: ubuntu-latest
+
+    timeout-minutes: 20
+
+    # https://github.com/bahmutov/cypress-gh-action-split-install/blob/ca3916d4e7240ebdc337825d2d78eb354855464b/.github/workflows/tests.yml#L8-L11
+    env:
+      # prevents extra Cypress installation progress messages
+      CI: 1
+      # avoid warnings like "tput: No value for $TERM and no -T specified"
+      TERM: xterm
+      # default to being a preview deployment
+      DEPLOY_DOCS_ENV: Preview
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup Node (uses version in .nvmrc)
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: '.nvmrc'
+
+      - name: Get yarn cache
+        id: yarn-cache
+        run: echo "::set-output name=dir::$(yarn config get cacheFolder)"
+
+      - uses: actions/cache@v3
+        with:
+          path: ${{ steps.yarn-cache.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+
+      - name: Install dependencies
+        run: yarn install --immutable
+        env:
+          # https://github.com/bahmutov/cypress-gh-action-split-install/blob/ca3916d4e7240ebdc337825d2d78eb354855464b/.github/workflows/tests.yml#L14-L18
+          # https://github.com/marketplace/actions/cypress-io#custom-install
+          CYPRESS_INSTALL_BINARY: 0
+
+      - name: Building docs website
+        run: yarn build:website
+
+      - name: Detect if is a production deployment (documentation)
+        if: github.event_name == 'push' && github.ref_name == 'main'
+        run: |
+          echo "DEPLOY_DOCS_ENV=Production" >> $GITHUB_ENV
+
+      - name: Deploy to Vercel (Production)
```

Comment:
I'm just curious about how does the Vercel step knows which folder we want to deploy since we don't specify it.
Does it automatically search for a `vercel-public` directory (we create it in the `yarn build:website` command) to deploy from?

Reply:
Yes it's defined in Vercel

<img width="805" alt="image" src="https://user-images.githubusercontent.com/1110551/167834819-33fb3a2e-dd46-4cbe-a64b-1a883d06ceae.png">